core/src/main/java/org/jdbi/v3/core/mapper/reflect/BeanMapper.java

```diff
@@ -89,8 +89,13 @@ protected PojoProperties<T> getProperties(ConfigRegistry config) {
         return (PojoProperties<T>) BeanPropertiesFactory.propertiesFor(type, config);
     }
 
-    @Override
+    @Deprecated // preserved for binary compatibility
```

Comment:
This gives me the error
```
WARNING: Incompatibility detected: Requires semantic version level	 MINOR
```
Should I bump the version as part of this MR? Since this is a feature and not a bugfix I would expect the minor version to be bumped anyways

Reply:
good question. That is a japi warning; does the build still succeed? We might sort that out at release time.

Reply:
`make install` fails and skips a lot of jobs. However `make tests` runs without any issues